README.md

```diff
@@ -162,3 +162,17 @@ directly the make targets). You can use Docker to run the test suite on non Linu
 ```console
 docker-compose run --rm test tox
 ```
+
```

Comment:
Can we move this out of the readme? I prefer to keep only user visible information here if possible

Reply:
The entire section? Where would you prefer to have it?

Reply:
Another file called MAINTAINANCE.md or something like that, but not in the readme, please